.pre-commit-config.yaml

```diff
@@ -47,20 +62,36 @@ repos:
       - id: flake8
         name: flake8 except __init__.py
         exclude: /__init__\.py$
+      #        additional_dependencies: ["flake8-bugbear==16.12.2"]
       - id: flake8
         name: flake8 only __init__.py
         args: ["--extend-ignore=F401"] # ignore unused imports in __init__.py
         files: /__init__\.py$
       #      - id: fix-encoding-pragma
       #        args: ["--remove"]
+      #        additional_dependencies: ["flake8-bugbear==16.12.2"]
       - id: check-case-conflict
       - id: check-docstring-first
       #      - id: check-executables-have-shebangs
       - id: check-merge-conflict
+        # exclude files where underlines are not distinguishable from merge conflicts
+        exclude: /README\.rst$|^docs/.*\.rst$
       - id: check-symlinks
       - id: check-xml
       - id: mixed-line-ending
         args: ["--fix=lf"]
+  #  - repo: https://github.com/pycqa/pylint
```

Comment:
Let's put the original standard configs, we should choose the correct pylint version for python2.7